sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb

```diff
@@ -204,6 +286,44 @@
     "# my_validation_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/vision-classification/valid\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3. Configure and run the AutoML for Images Classification-Multilabel training job\n",
+    "In this section we will configure and run the AutoML job, for training the model.\n",
+    "\n",
+    "## 3.1 Configure the job through the image_classification_multilabel() factory function\n",
+    "\n",
+    "### image_classification() function parameters:\n",
+    "\n",
+    "The `image_classification_multilabel()` factory function allows user to configure the training job.\n",
+    "\n",
+    "- `compute` - The compute on which the AutoML job will run. In this example we are using a compute called 'gpu-cluster' present in the workspace. You can replace it any other compute in the workspace.\n",
+    "- `experiment_name` - The name of the experiment. An experiment is like a folder with multiple runs in Azure ML Workspace that should be related to the same logical machine learning experiment.\n",
+    "- `name` - The name of the Job/Run. This is an optional property. If not specified, a random name will be generated.\n",
+    "- `target_column_name` - The name of the column to target for predictions. It must always be specified. This parameter is applicable to 'training_data', 'validation_data' and 'test_data'.\n",
```

Comment:
Please remove 'test_data' from this line. Image tasks don't support test_data yet.